### PR TITLE
[Service] deleting activated model

### PR DESCRIPTION
**Pull request by jaeyun-jung**
Add util functions in service-db,
1. prevent deleting activated model,
2. add util function to check model registration.

### CODE REVIEW

**Comment by anyj0527** [comment]
Need to change fix the unittest / TCT. There is an usecase of deleting active model.

**Comment by jaeyun-jung** [comment]
@anyj0527 updated. plz review the changes again.

**Review by songgot — APPROVED** [review]
Looks good to me.

File: tests/capi/unittest_capi_service_agent_client.cc [hunk 1]
```diff
@@ -1154,14 +1154,11 @@ TEST_F (MLServiceAgentTest, model_scenario)
...
   g_free (info_list);
 
-  /* delete the active model */
+  /* failed to delete the active model */
   status = ml_service_model_delete (key, 1U);
-  EXPECT_EQ (ML_ERROR_NONE, status);
-
-  /* no active model */
-  status = ml_service_model_get_activated (key, &activated_model_info);
   EXPECT_EQ (ML_ERROR_INVALID_PARAMETER, status);
```
Comment (again4you) [inline tests/capi/unittest_capi_service_agent_client.cc:1159]: ### Question
In this case, which error code is much better `ML_ERROR_PERMISSION_DENIED` or `ML_ERROR_INVALID_PARAMETER`?

**Review by again4you — APPROVED** [review]
Except simple opinion, this code is good to me!

**Pull request merged**